portality/forms/application_forms.py

```diff
@@ -1626,7 +1626,10 @@ class FieldDefinitions:
                                 "the journal must use a persistent identifier"
                 }
             }
-        ]
+        ],
+        "widgets": [
+            "article_info",
+        ],
```

Comment:
Probably worth adding a comment here that the article_info back-end is only accessible to admins.  The DOAJ_SEAL also only appears in the admin form context, so that's all good, but I suppose we could change that at some point and not notice.